corehq/apps/locations/templates/locations/manage/location.html

```diff
@@ -83,6 +83,25 @@
       </script>
 
       {% if not form.is_new_location %}
+
+        <p class="lead">{{ location.name }} <small>{{ location.location_type_name }}</small></p>
```

Comment:
ah i see. i agree some sort of separator would be helpful. i read it as Duval County not Duval is of type County